### PR TITLE
Handle SCC

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by phoracek** [comment]
@SchSeba could you please take a look?

File: data/multus/001-rbac.yaml [hunk 1]
```diff
@@ -36,3 +36,19 @@ kind: ServiceAccount
...
   namespace: multus
+{{ if .EnableSCC }}
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: multus
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostNetwork: true
```
Comment (booxter) [inline data/multus/001-rbac.yaml]: why do we need host-network for multus? AFAIU the container merely copies a binary to host but plugin is executed outside the container.

File: data/linux-bridge/001-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  name: linux-bridge
+  namespace: linux-bridge
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: linux-bridge
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostNetwork: true
```
[the remark this answers is not in the document]
Comment (phoracek) [reply data/linux-bridge/001-rbac.yaml]: There is no marker yet. I think it is there only so the container can be used before the primary network is created. That however won't be case with our operator that requires networking to be already in place.

The same for Multus. Will try to drop both.

**Comment by phoracek** [comment]
@booxter fixed, could you take another look?

**Comment by phoracek** [comment]
@SchSeba I'd prefer not to create an unused account

**Comment by SchSeba** [comment]
@phoracek no problem :)

so from my side it's good to go

**Comment by SchSeba** [comment]
did you try to apply an empty yaml?

**Comment by phoracek** [comment]
@SchSeba I did, empty yaml is ignored by the renderer. I tried this on both openshift and kubernetes, both worked.

**Pull request merged**